spec/observables/timer-spec.js

```diff
@@ -11,4 +11,27 @@ describe('Observable.timer', function () {
 
     expectObservable(source).toBe(expected, {x: 0});
   });
-});
\ No newline at end of file
+
+  it('should emit a single value immediately', function () {
+    var source = Observable.timer(0, rxTestScheduler);
+    var expected = '(x|)';
+
+    expectObservable(source).toBe(expected, {x: 0});
+  });
+
+  it('should start after delay and periodically emit values', function () {
+    var source = Observable.timer(40, 20, rxTestScheduler).take(5);
+    var expected = '----a-b-c-d-(e|)';
+    var values = { a: 0, b: 1, c: 2, d: 3, e: 4};
+
+    expectObservable(source).toBe(expected, values);
+  });
+
+  it('should start immediately and periodically emit values', function () {
```

Comment:
Sure! Much better this way. I was not aware of the second argument of `expectObservable()`.

```javascript
it('should stop emiting values when subscription is done', function () {
  var source = Observable.timer(0, 30, rxTestScheduler);
  var expected = 'a--b--c--d--e';
  var unsub   =  '             !'
  var values = { a: 0, b: 1, c: 2, d: 3, e: 4};

  expectObservable(source, unsub).toBe(expected, values);
});
```

But I 'm not sure if we can use `expectSubscriptions()` here since the observable returned by `timer()` does not have a subscription log.